guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

```diff
@@ -28,10 +28,10 @@ You can create additional organizations and then import additional manifests to
 endif::[]
 
 .Multi-homed {Project} and {SmartProxy}
-If you want {Project} to use multiple interfaces with distinct host names, you must perform additional configuration of the {ProjectServer} and {Project} {SmartProxyServer} CA certificates.
-ifeval::["{build}" == "satellite"]
-If you want to deploy {Project} in this configuration, contact Red Hat.
-endif::[]
+
+Multi-homed {Project} is not supported.
```

Comment:
I would keep this as a Satellite specific note. Support status is really a downstream only thing.